### DISCUSSION TITLE
Can I hide the filer UI

### DISCUSSION THREAD

**Discussion by dc-vin** [seaweedfs/seaweedfs#3717] [Q&A] [answered]
For security reasons, the page will be hidden, or the login mechanism will be embedded

**Comment by aoberest** [comment, marked as answer]
Hello
You can use JwtToken
https://github.com/seaweedfs/seaweedfs/wiki/Security-Overview#securing-filer-http-with-jwt

**Reply by dc-vin** [reply]
Thank you for your help
